solidity/test/TokenHolder.js

```diff
@@ -1,52 +1,179 @@
 const { accounts, contract } = require('@openzeppelin/test-environment');
 const { expect } = require('../../chai-local');
-const { expectRevert, constants, BN } = require('@openzeppelin/test-helpers');
+const { expectRevert, constants, BN, balance } = require('@openzeppelin/test-helpers');
 
 const { ZERO_ADDRESS } = constants;
 
+const { NATIVE_TOKEN_ADDRESS } = require('./helpers/Constants');
+
 const TokenHolder = contract.fromArtifact('TokenHolder');
 const TestStandardToken = contract.fromArtifact('TestStandardToken');
 
 describe('TokenHolder', () => {
+    const getBalance = async (tokenAddress, account) => {
+        if (tokenAddress === NATIVE_TOKEN_ADDRESS) {
+            return balance.current(account);
+        }
+
+        return (await TestStandardToken.at(tokenAddress)).balanceOf.call(account);
```

Comment:
Watch out, we need two separate awaits here - one for the `at` (starting from web3.js v1.x), and one for the `balanceOf.call`.

Reply:
Same as above